src/panel/info.ts

```diff
@@ -21,6 +23,138 @@ export class InfoPanel {
 
         this.root.addChild(rect);
 
+        this.infoBox = new InfoBox(this);
+
         parent.addChild(this.root);
     }
+}
+
+
+export class InfoBox {
+    readonly root: Container;
+    readonly width: number;
+    readonly height: number;
+
+    readonly infoContainer: Container;
+
+    constructor(parent: InfoPanel) {
+        this.root = new PIXI.Container();
+        this.root.position.set(0, 0);
+        
+        this.width = 0.9 * parent.width;
+        this.height = 0.4 * parent.height;
+
+        let x = parent.width * 0.05;
+        let y = parent.height * 0.95 - this.height;
+        const rect = new PIXI.Graphics()
+            .lineStyle(5)
+            .beginFill(Colors.LIGHT_BLUE_WHITE)
+            .drawRoundedRect(x, y, this.width, this.height, 45)
+            .endFill()
+        this.root.addChild(rect)
+
+        // Its this classes responsibility to dispose of old information
+        // thus it manages the sub-container
+        this.infoContainer = new PIXI.Container();
+        this.infoContainer.position.set(x, y);
+        this.root.addChild(this.infoContainer)
+
+        parent.root.addChild(this.root);
+    }
+
+    /**
+     * Update the info box with a new container. 
+     */
+    public updateInfo_raw(info: Container) {
+        this.infoContainer.removeChildren();
+        this.infoContainer.addChild(info);
+    }
+
+    /**
+     * A helper method for updating the info box
+     */
+    public updateInfo_color_pos_text_header(color: ColorSource, pos: Point, headerText: string, bodyText: string, headerStyle?: Partial<ITextStyle>, bodyStyle?: Partial<ITextStyle>) {
```

Comment:
I'm going to complain about your mixing of `camelCase` and `snake_case` here. Moreover imo these don't have reasons to be overloads, this is just one public API method and one private method, which could be `public updateInfo` and `private setInfo` or something like that

Reply:
I didnt want to lock you out from just putting there what ever you thought you needed. But also provide easier methods in case you didnt wanna go in and design it yourself